src/components/graphic.js

```diff
@@ -1,11 +1,15 @@
 import React from "react";
 import { GridContainer, Grid } from "@trussworks/react-uswds";
 
-const Graphic = () => {
+const Graphic = ({ images, graphicHeadings }) => {
   return (
     <GridContainer>
-      <Grid col>
-        <Grid></Grid>
+      <Grid row>
+        {images.map((image) => (
+          <Grid col={3}>
+            <p>{image}</p>
```

Comment:
What is `image`? If it's actually an image, do you want to put it in between a `<p>` tag? If it's text and not an image, I would suggest using a more descriptive variable name (i.e. `imageText`).

Reply:
I think you're not commenting on the most up to date version. I was just doing this to see if the content was coming through. It should be an img tag now